### DISCUSSION TITLE
Wireguard Client IP hidden behind PiVPN's host IP - Pihole reports activity based on pivpn's host IP but not the wireguard client IP

### DISCUSSION THREAD

**Discussion by tommyboy180** [pivpn/pivpn#1742] [Support]
My wireguard IPs are in the 10.230.122.x subnet but my PiHole (192.168.50.3) only sees traffic from the PiVPN host (192.168.50.164)

Several clients appear as a single device. Is there a way for IPTABLES to forward the wireguard client IPs rather than hiding behind the pivpn's host IP?

![image](https://github.com/pivpn/pivpn/assets/565704/fca30ce9-7b97-4ace-9c94-284456e65abb)

**Comment by orazioedoardo** [comment]
Sorry for the delay, that's expected due to the default config using NAT for maximum compatibility. If you want to see 10.230.122.0/24 IPs in the Pi-hole log, you need to disable NAT, see: https://github.com/pivpn/pivpn/issues/1014

**Reply by coolapso** [reply]
should that be added to documentation?

**Reply by orazioedoardo** [reply]
Perhaps yes although it's just easier to run pivpn and pihole on the same host.
